lib/widgets/subscription_list.dart

```diff
@@ -230,7 +235,8 @@ class SubscriptionItem extends StatelessWidget {
 
     final swatch = colorSwatchFor(context, subscription);
     final hasUnreads = (unreadCount > 0);
-    final opacity = subscription.isMuted ? 0.55 : 1.0;
+    const mutedOpacity = 0.55;
```

Comment:
It would be helpful to link the source of the opacity.

Reply:
I am not sure if I got it correctly; I just used the same one that we've used before for the channel name and unread count badge.

Are you suggesting to link it to zulip mobile or web design?

Reply:
Oh I see. Yeah `0.55` is not new here; it was added in b0b8a50703d2633fbc629a680f4f58ac4a6e8251. The design came from Vlad [here](https://github.com/zulip/zulip-flutter/issues/424#issuecomment-1831121034). Perhaps we can link to that GitHub comment in an NFC commit.